src/Starlight/Game.hs

```diff
@@ -166,7 +189,9 @@ game = Sol.system >>= \ system -> runGame system $ do
 
   runFrame . runReader UI{ fps = fpsLabel, target = targetLabel, face } . fix $ \ loop -> do
     measure "frame" frame
-    measure "swap" Window.swap *> loop
+    measure "swap" Window.swap
+    loop
+  put True
```

Comment:
We use this boolean variable to communicate that the background thread should halt.

In normal operation the exit of the `main` thread terminates the process, but in `ghci` the background thread will just keep spinning forever otherwise.